docs/en/install-upgrade/air-gapped-install.asciidoc

```diff
@@ -0,0 +1,532 @@
+
+The Elastic Kubernetes operator is an additional component in the Kubernetes OpenShift install that, essentially, does a lot of the work in installing, configuring, and updating deployments of the {stack}. For details, refer to the {eck-ref}/k8s-air-gapped.html[{eck} install instructions].
+
+The main requirements are:
+
+* Syncing container images for ECK and all other {stack} components over to a locally-accessible container repository.
+* Modifying the ECK helm chart configuration so that ECK is aware that it is supposed to use your offline container repository instead of the public Elastic repository.
+* Optionally, disabling ECK telemetry collection in the ECK helm chart. This configuration propagates to all other Elastic components, such as {kib}.
+* Building your custom deployment container image for the {artifact-registry}.
+* Building your custom deployment container image for the Elastic Endpoint Artifact Repository.
+
+[discrete]
+[[air-gapped-k8s-os-elastic-package-registry]]
+==== 2.2. Elastic Package Registry
+
+The container image can be downloaded from the official Elastic Docker repository, as described in the {fleet} and {elastic-agent} {fleet-guide}/air-gapped.html[air-gapped environments] documentation.
+
+This container would, ideally, run as a Kubernetes deployment. Refer to <<air-gapped-epr-kubernetes-example>> for examples.
+
+[discrete]
+[[air-gapped-k8s-os-elastic-artifact-registry]]
+==== 2.3. {artifact-registry}
+
+A custom container would need to be created following similar instructions to setting up a web server in the <<air-gapped-elastic-artifact-registry,self-managed install case>>. For example, a container file using an NGINX base image could be used to run a build similar to the example described in <<air-gapped-elastic-artifact-registry-example>>.
+
+[discrete]
+[[air-gapped-k8s-os-elastic-endpoint-artifact-repository]]
+==== 2.4. Elastic Endpoint Artifact Repository
+
+Just like the {artifact-registry}. A custom container needs to be created following similar instructions to setting up a web server for the <<air-gapped-elastic-artifact-registry,self-managed install case>>.
+
+[discrete]
+[[air-gapped-k8s-os-ironbank-secure-images]]
+==== 2.5. Ironbank Secure Images for Elastic
+
+Besides the public link:https://www.docker.elastic.co[Elastic container repository], most {stack} container images are also available in Platform One's link:https://ironbank.dso.mil/repomap?vendorFilters=Elastic&page=1&sort=1[Iron Bank].
+
+[discrete]
+[[air-gapped-elastic-package-registry-example]]
+=== Appendix A - {package-registry}
+
+The following script generates a SystemD service file on a RHEL 8 system in order to run EPR with Podman in a production environment.
+
+[source,shell]
+----
+#!/usr/bin/env bash
+
+EPR_BIND_ADDRESS="0.0.0.0"
+EPR_BIND_PORT="8443"
+EPR_TLS_CERT="/etc/elastic/epr/epr.pem"
+EPR_TLS_KEY="/etc/elastic/epr/epr-key.pem"
+EPR_IMAGE="docker.elastic.co/package-registry/distribution:8.4.3"
```

Comment:
This contains a fixed version number. Can we make this to the same one as the page you are viewing?